docusaurus/docs/React/message-components/attachment.mdx

```diff
@@ -103,6 +106,56 @@ const CustomAttachment = (props) => {
 </Chat>;
 ```
 
+### Image and video sizing
+
+The following section details how the width and height of images and videos uploaded from files are computed.
+
+#### Maximum size
+
+You can control the maximum size of images and videos with the [`--str-chat__attachment-max-width`](../theming/component-variables.mdx) CSS variable (available only in [theme-v2](../theming/introduction.mdx)). The value of this variable must be a value that can be computed to a valid pixel value using the [`getComputedStyle`](https://developer.mozilla.org/en-US/docs/Web/API/Window/getComputedStyle) method (for example: `300px`, `10rem`, `calc(300px - var(--margin))`, but not `100%`). If you provide an invalid value, the image and video sizing can break, which can lead to scrolling issues inside the message list (for example the message list isn't scrolled to the bottom when you open a channel).
+
+If you set an invalid value to the variable, you'll see a warning on the browser's console:
+
+<img src={AttachmentSizeWarning} width='800' />
+
+#### File size optimization
+
+Based on the CSS settings provided for images and videos, the SDK will load an image (or thumbnail image in case of a video file) with a reduced file size while still providing sufficient image quality for the given dimensions. This will result in less network traffic and faster image load for users.
```

Comment:
Without reading the source code, it was not clear to me, how the configuration is provided / how it works.

Reply:
Thanks, I added some clarifications, I decidedly not include implementation details as it should be important to integrators, I just wanted to let them know that this feature exists. Let me know if you think it needs adjusting